src/components/modeler/Modeler.vue

```diff
@@ -220,6 +221,9 @@ export default {
     },
   },
   methods: {
+    saveBpmn() {
+      this.$emit('saveBpmn');
```

Comment:
Since this just re-emits the event would it be possible to do that inline in the handler listed in the template?